packages/bun-types/bun.d.ts

```diff
@@ -6287,7 +6343,7 @@ declare module "bun" {
     /**
      * TLS Configuration with which to create the socket
      */
-    tls?: boolean;
+    tls?: TLSOptions | boolean;
```

Comment:
Add to JSDoc why `boolean` is supported here/highlight the difference between the object or just a boolean